### DISCUSSION TITLE
Not all values copied.

### DISCUSSION THREAD

**Discussion by betterapp** [pimcore/pimcore#13601] [Q&A / Support]
Hi. I have a Product class which is very complex.
Have relation to bricks, other classes, fields collection and allow inheritance.

I have a product tree. The child products inherited values from parent in many cases.

When I try to copy the product out of the tree to different location I see that not all values
are copied. Some fields are empty.

Why ? I can not find anything about it in documentation.

**Comment by mattamon** [comment]
Hey @betterapp,

what type of fields are empty?
Objectbricks, fieldcollections, relations? 

Can you maybe provide an example?

**Comment by betterapp** [comment]
@mattamon 

Basically for all fields type that allow inheritance.
For example if we have a object tree and object allow inheritance some of the values in child object are taken from the parent object.
If we copy  lowest level object and move it out of the tree to different location then the value from inherited fields are not set in copied object.
